maps/biter_battles_v2/main.lua

```diff
@@ -68,6 +71,39 @@ local function on_entity_died(event)
 	Game_over.silo_death(event)
 end
 
+local function getTagOutpostName(pos)
+	if pos < 0 then
+		return autoTagWestOutpost
+	else
+		return autoTagEastOutpost
+	end
+end
+
+local function autotagging_outposters()
+    for _, p in pairs(game.connected_players) do
+		if p and p.valid then
+		
+			if (p.force.name == "north" or p.force.name == "south") then
+				if math.abs(p.position.x) < autoTagDistance then
+					if string.find(p.tag, '%'..autoTagWestOutpost) or string.find(p.tag, '%'..autoTagEastOutpost) then
+						p.tag = p.tag:gsub("%"..autoTagWestOutpost, "")
+						p.tag = p.tag:gsub("%"..autoTagEastOutpost, "")
+					end
+				else
+					if not string.find(p.tag, '%'..autoTagWestOutpost) and not string.find(p.tag, '%'..autoTagEastOutpost) then
+						p.tag = p.tag .. getTagOutpostName(p.position.x)
+					end
+				end
+			end
+			
+			if p.force.name == "spectator" and (string.find(p.tag, '%'..autoTagWestOutpost) or string.find(p.tag, '%'..autoTagEastOutpost)) then
```

Comment:
```suggestion
			if p.force.name == "spectator" and string.find(p.tag, 'Outpost') then
```
nitpick: either one is good

Reply:
Okay finally I commited it, but after thinking I disagree on it, what if someone made a group [laserOutpost], okay it's very specific and will barely happen but I think we should catch the exact name.

If you still disagree on it and want not exact name, then sure I will do what you suggested.